### DISCUSSION TITLE
terser extractComments are wrapped in a comment (comment in wrapped comment)

### DISCUSSION THREAD

**Discussion by Venipa** [webpack/webpack#15132] [Q&A]
i've a file with a userscript banner and im passing it with a raw property but it still gets wrapped in a comment which breaks my userscript recognition.

Example (my banner txt)
```
// ==UserScript==
// @name         Userscript-Name
// ==/UserScript==
```

gets turned into
```
/*! // ==UserScript==
// @name         Userscript-Name
// ==/UserScript== */
```

### Reproduce

Steps:

- git clone https://github.com/Venipa/coubdl-bridge-userscript
- yarn
- yarn prod

Then check dist/*.user.js file
Banner content is in assets folder

**Comment by alexander-akait** [comment]
Can you provide reproducible example?

**Reply by Venipa** [reply]
Steps:
- git clone https://github.com/Venipa/coubdl-bridge-userscript
- yarn
- yarn prod

Then check dist/*.user.js file
Banner content is in assets folder

**Reply by alexander-akait** [reply]
Yep, because we have logic https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/src/index.js#L647, it was design for other goals, why do not use https://webpack.js.org/plugins/banner-plugin/ to inject banner, it was design for this goals?
